File: bubble/bubbleSort.js
let numbers = [8,2,6,4,9,5,1,3,7,4,5,6];

function bubbleSort(array) {
    for (let i = 0; i < array.length; i++) {
        for (let j = 0; j < array.length - 1; j++) {
            if (array[j] > array[j+1]) {
                let e = array[j];
                array[j] = array[j+1];
                array[j+1] = e;
            }
        }
    }
    return array;
}

let sorted = bubbleSort(numbers);
console.log(sorted);

File: bubble/bubbleSort.py
def main():
    numbers = [8,2,6,4,9,5,1,3,7,4,5,6]
    bubbleSort(numbers, len(numbers))
    print(numbers)

def bubbleSort(anArray, length):
    for i in range(length):
        for j in range(length - 1):
            if anArray[j] > anArray[j+1]:
                n = anArray[j]
                anArray[j] = anArray[j+1]
                anArray[j+1] = n

if __name__=="__main__":
    main()